### PR TITLE
Dwm mc ks/lookup registry at run time

### PR DESCRIPTION
**Pull request by ksubrama**
Also, enable some basic serverspec tests on windows.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -13,12 +13,21 @@ platforms:
   - name: ubuntu-14.04
     run_list:
       - recipe[apt::default]
-
+  - name: windows-2012r2
```
Comment (schisamo) [inline .kitchen.yml]: Please add the following comment above the private things:
https://github.com/chef-cookbooks/languages/blob/master/.kitchen.yml#L27-L35

File: .kitchen.yml [hunk 2]
```diff
@@ -13,12 +13,21 @@ platforms:
...
+    driver:
+      box: chef/windows-server-2012r2-standard
```
Comment (schisamo) [inline .kitchen.yml]: Mark this box private:

```yaml
box: chef/windows-server-2012r2-standard # private
```

File: test/integration/windows/serverspec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'serverspec'
+require 'pathname'
+
+set :backend, :cmd
+set :os, family: 'windows'
```
Comment (schisamo) [inline test/integration/windows/serverspec/spec_helper.rb]: Move to `test/integration/helpers/serverspec/spec_helper` and the spec_helper will automatically be shared by all suites. Example here:
https://github.com/chef-cookbooks/languages/blob/master/test/integration/helpers/serverspec/spec_helper.rb

**Comment by schisamo** [comment]
:+1: 

Thanks for addressing all my comments @ksubrama!

**Comment by manderson26** [comment]
LGTM :+1: Thanks for adding test kitchen stuff.

**Pull request merged**